.babelrc

```diff
@@ -1,11 +1,20 @@
 {
 	"presets": [
-		"es2015",
+		[ "env", {
+			"targets": {
+				"browsers": [
+					"last 2 versions",
+					"Safari >= 9",
+					"iOS >= 9",
+					"not ie <= 10",
```

Comment:
`last 2 versions` + `ie 11` would include `ie 10`, because the rules are implicitly joined by a logical `OR` (`not ...` is an exception).